lark/parser_frontends.py

```diff
@@ -22,19 +24,15 @@ def lex(self, lexer_state, parser_state):
         return CustomLexerWrapper
 
 
-class MakeParsingFrontend:
-    def __init__(self, parser_type, lexer_type):
-        self.parser_type = parser_type
-        self.lexer_type = lexer_type
+def _deserialize_parsing_frontend(data, memo, lexer_conf, callbacks, options):
```

Comment:
`MakeParsingFrontend.deserialize()` wasn't using the instance variables, so it could be converted into a standalone function.